### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by zwcloud**
Fix links for dotnet-apiport.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/porting/index?branch=pr-en-us-1163](https://review.docs.microsoft.com/en-us/dotnet/articles/core/porting/index?branch=pr-en-us-1163)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C20%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610200425405778-1163%5Cworkflow_report.html).

**Review by mairaw — CHANGES_REQUESTED** [review]
HI @zwcloud, thanks for your contribution! From what I can see in one of their [commits](https://github.com/Microsoft/dotnet-apiport/commit/678f6c0ad9c19096fdc806d09c263ee08a10110d), the content from that file was moved over to the readme file. So can you please just update the links to point to https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/ instead of the repo homepage? Thanks!

File: docs/core/porting/index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The recommended process for porting follows the following series of steps.  Each
 
    This ensures that you can use API alternatives for .NET Framework-specific targets in the cases where .NET Core can't support a particular API.
    
-3. Use the [API Portability Analyzer tool](https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/Introduction.md) to analyze your assemblies and develop a plan to port based on its results.
+3. Use the [API Portability Analyzer tool](https://github.com/Microsoft/dotnet-apiport/) to analyze your assemblies and develop a plan to port based on its results.
```
Comment (mairaw) [inline docs/core/porting/index.md]: I think this one is fine as-is. No need to change this one.

File: docs/core/porting/index.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The recommended process for porting follows the following series of steps.  Each
...
-* Api Portability Analyzer - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b), a toolchain that can generate a report of how portable your code is between .NET Framework and .NET Core, with an assembly-by-assembly breakdown of issues.  See [Tooling to help you on the process](https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/Introduction.md) for more information.
+* Api Portability Analyzer - [command line tool](https://github.com/Microsoft/dotnet-apiport/releases) or [Visual Studio Extension](https://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b), a toolchain that can generate a report of how portable your code is between .NET Framework and .NET Core, with an assembly-by-assembly breakdown of issues.  See [Tooling to help you on the process](https://github.com/Microsoft/dotnet-apiport/) for more information.
```
Comment (mairaw) [inline docs/core/porting/index.md]: Let's change this one to 
```markdown
[.NET Portability Analyzer Tools](https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/)
```

Comment (zwcloud) [inline docs/core/porting/index.md]: Yes, that's more approprate. Updated.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/core/porting/index?branch=pr-en-us-1163](https://review.docs.microsoft.com/en-us/dotnet/articles/core/porting/index?branch=pr-en-us-1163)  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C22%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610221310181217-1163%5Cworkflow_report.html).

File: docs/core/porting/index.md (hunk 1 above)
Comment (mairaw) [inline docs/core/porting/index.md]: I didn't express myself correctly. Yes, the change is fine as is.

**Review by mairaw — APPROVED** [review]

**Pull request merged**